apps/container/src/views/HistoryView.vue

```diff
@@ -283,47 +332,35 @@ const extensionURL =
 
 const studentRecordURL = 'https://aluno.ufabc.edu.br/fichas_individuais';
 
-const conceptsColor = {
-  A: 'rgb(63, 207, 140)',
-  B: 'rgb(184, 233, 134)',
-  C: 'rgb(248, 183, 76)',
-  D: 'rgb(255, 160, 4)',
-  F: 'rgb(249, 84, 105)',
-  O: 'rgb(169, 169, 169)',
-
-  // exceptions
-  I: 'rgb(25, 118, 210)',
-  E: 'rgb(25, 118, 210)',
-  null: 'rgb(0, 0, 0)',
-};
-
 const subjectConceptClass = {
   A: 'gray',
   B: 'gray',
   C: 'gray',
   D: 'gray',
   O: 'error',
   F: 'error',
+  E: 'error',
+  I: 'error',
 } satisfies Record<Concept, string>;
 
 const {
-  data: enrollment,
+  data: enrollments,
   isLoading: isLoadingEnrollment,
-  error: errorEnrollment,
+  isError: errorEnrollment,
```

Comment:
aqui faz sentido mandar pro plural o error e o loading, também colocar o is antes do error para não parecer que é o erro que foi obtido